test/JDBC/upgrade/13_4/preparation/case_insensitive_collation-before-13-5-vu-prepare.sql

```diff
@@ -0,0 +1,38 @@
+-- BABEL-3094
```

Comment:
we don't need maintain this file under 13_4 and 13_5 folder. we can put them under common folder input/ and add entry in 13_4/schedule and 13_5/schedule file.

Reply:
done